test/src/test/java/org/corfudb/runtime/view/stream/BackpointerStreamViewTest.java

```diff
@@ -113,4 +111,30 @@ public void moreReadQueueTest() {
             assertThat(new String(payLoad).equals(String.valueOf(i)));
         }
     }
+
+    /**
+     *  test proper backpointer termination at the head of a stream
+     *
+     * */
+    @Test
+    public void headOfStreamBackpointerTermination() {
+
+        final int totalEntries = 101;
```

Comment:
Do you mean that totalEntries is the same as PARAMETERS.NUM_ITERATIONS_LOW? then use that instead of a constant. Note that PARAMETERS gets different values inside travis.